crates/telemetry-subscribers/Cargo.toml

```diff
@@ -8,6 +8,10 @@ repository = "https://github.com/mystenlabs/mysten-infra"
 edition = "2021"
 publish = false
 
+[[bin]]
+name = "import-trace"
+path = "src/bin/import_trace.rs"
+
```

Comment:
If you rename the file below you could technically remove this from the toml file.

I suppose that's up to personal preference though